sites/public/src/components/listing/ListingView.tsx

```diff
@@ -150,7 +156,11 @@ export const ListingView = (props: ListingProps) => {
     buildingSelectionCriteria = (
       <p>
         <a
-          href={fileService.getDownloadUrlForPdf(listing.buildingSelectionCriteriaFile.fileId)}
+          href={getImageUrlFromAsset(
```

Comment:
Should this be pdf instead since it's a pdf file?

Reply:
Good catch!  Fix inbound